application/features/bootstrap/EmailContext.php

```diff
@@ -905,4 +911,63 @@ public function iSeeThatTheSecondUserHasReceivedAGetBackupCodesEmail()
     {
         Assert::true($this->fakeEmailer->hasReceivedMessageRecently($this->tempUser2->id, EmailLog::MESSAGE_TYPE_GET_BACKUP_CODES));
     }
+
+    /**
+     * @Given no methods exist
+     */
+    public function noMethodsExist()
+    {
+        Method::deleteAll();
+    }
+
+    /**
+     * @param string $value
+     */
+    protected function createMethod($value)
+    {
+        $user = $this->tempUser;
+        $method = Method::findOrCreate($user->id, $value);
+
+        $this->testMethod = $method;
+
+        Assert::true($method->save(), "Could not create new method.");
+    }
+
+    /**
+     * @When I create a new recovery method
+     */
+    public function iCreateANewRecoveryMethod()
+    {
+        $this->createMethod(self::METHOD_EMAIL_ADDRESS);
+    }
+
+    /**
+     * @Then a Method Verify email is sent to that method
+     */
+    public function aMethodVerifyEmailIsSentToThatMethod()
+    {
+        $matchingFakeEmails = $this->fakeEmailer->getFakeEmailsOfTypeSentToUser(
+            EmailLog::MESSAGE_TYPE_METHOD_VERIFY, self::METHOD_EMAIL_ADDRESS);
```

Comment:
Again a very minor issue, but some PSR-2 checker would probably complain about this since the arguments to the function were long enough to wrap, but weren't each on their own separate line.
```suggestion
            EmailLog::MESSAGE_TYPE_METHOD_VERIFY,
            self::METHOD_EMAIL_ADDRESS
        );
```